### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by LooseScrew01** [Paperback-iOS/app#840]
### Preflight Checklist

- [X] I have read and I agree to follow the [support disclaimer](https://github.com/Paperback-iOS/app#support-disclaimer).
- [X] I am sure that my bug has nothing to with third party extensions.
- [X] I have read the [troubleshooting FAQs](https://paperback.moe/help/faq/) and looked trough the \#help-desk channel on the [official Discord](https://discord.com/invite/Paperback), but none solved my issue.
- [X] I have searched the [issue tracker](https://github.com/Paperback-iOS/app/issues) for a bug report that matches the one I want to file, without success.

### What type of Apple device have you encountered the bug with?

iPhone

### Operating System Version

iOS 17.2.1

### App Version

Newest version as of this post

### Describe the Bug

The app instantly crashes when trying to open

### Steps to Reproduce

I was downloading The Legend of the Northern Blade and set my phone aside with the download manager tab open watching the MBs, ive downloaded ~10 mangas and the downloads sometimes halt alone, i fix it by reloading the app, i did this with my newest download a few times and it was normal, then it just wouldnt open, i press it and 0.1 seconds later it crashes

**Comment by Celarye** [comment]
@LooseScrew01 Try a database repair, this tends to fix this issue. And for in the future, please fill in the form correctly, a title and version number is missing.

**Comment by LooseScrew01** [comment]
> @LooseScrew01 Try a database repair, this tends to fix this issue. And for in the future, please fill in the form correctly, a title and version number is missing.

How do i do database repair? Sorry for not filling correctly

**Comment by Celarye** [comment]
> How do i do database repair? Sorry for not filling correctly

You can find it on the Settings page.

**Comment by LooseScrew01** [comment]
> > How do i do database repair? Sorry for not filling correctly
> 
> You can find it on the Settings page.

I cant open the app, thats the issue

**Comment by Celarye** [comment]
> I cant open the app, thats the issue

Not at all? Does it instantly crash? In this case it can still be fixed but you'll need to have access to a computer.

**Comment by LooseScrew01** [comment]
> > I cant open the app, thats the issue
> 
> Not at all? Does it instantly crash? In this case it can still be fixed but you'll need to have access to a computer.

Yes it instantly crashes, i have access to a computer

**Comment by Celarye** [comment]
Yes, there are some things we can try, I’ll write a step by step guide once I get home.

**Comment by LooseScrew01** [comment]
> Yes, there are some things we can try, I’ll write a step by step guide once I get home.

Ok thank you i really appreciate it

**Comment by Celarye** [comment]
You can try removing the extensions manually using iTunes on Windows or Finder on macOS:
1. Close the app
2. Connect your phone to your pc
3. Open Finder/iTunes
4. Go to the files tab of your connected device
5. Remove the "extensions" folder
6. Disconnect your phone and try to start the app

Let me know if this works

**Comment by LooseScrew01** [comment]
> You can try removing the extensions manually using iTunes on Windows or Finder on macOS:
> 
> 1. Close the app
> 2. Connect your phone to your pc
> 3. Open Finder/iTunes
> 4. Go to the files tab of your connected device
> 5. Remove the "extensions" folder
> 6. Disconnect your phone and try to start the app
> 
> Let me know if this works

Didnt work
![image](https://github.com/Paperback-iOS/app/assets/99226098/db2ce2ca-2562-4b1f-b8d2-4822ce9f1c3a)

**Comment by Celarye** [comment]
@rintendou anything else he can try?

**Comment by LooseScrew01** [comment]
> @rintendou anything else he can try?

Thank you for your help

**Comment by rintendou** [comment]
Try copying the default.realm and doing a fresh install. Then do the same steps and drag the default.realm in.

**Comment by LooseScrew01** [comment]
> Try copying the default.realm and doing a fresh install. Then do the same steps and drag the default.realm in.

Can you explain in layman terms please

**Comment by Celarye** [comment]
You can see a file named `default.realm` in the screenshot you send.

1. Close the app.
2. Connect your phone to your computer.
3. Open the files tab for this app in iTunes.
4. Drag the `default.realm` file to your desktop.
5. Disconnect your device from your computer.
6. Reinstall the app.
7. Reconnect your phone to your computer.
8. Open the files tab for this app again.
9. Remove the new `default.realm` file by dragging it to your desktop as well (but don't replace the old one by doing this, give it another name).
10. Drag the old one back into the files tab of the app.
11. Disconnect your device and try to launch the app.

**Comment by LooseScrew01** [comment]
> You can see a file named `default.realm` in the screenshot you sent.
> 
> 1. Close the app.
> 2. Connect your phone to your computer.
> 3. Open the files tab for this app in iTunes.
> 4. Drag the `default.realm` file to your desktop.
> 5. Disconnect your device from your computer.
> 6. Reinstall the app.
> 7. Reconnect your phone to your computer.
> 8. Open the files tab for this app again.
> 9. Remove the new `default.realm` file by dragging it to your desktop as well (but don't replace the old one by doing this, give it another name).
> 10. Drag the old one back into the files tab of the app.
> 11. Disconnect your device and try to launch the app.

I have a couple questions, when I drag the files they get copied to the desktop not moved, I tried both ways though first I copied and left them, and followed the steps, second time I copied and deleted from the phone files still didn't work, my second question is, when you say reinstall the app, I just offloaded it and reinstalled it, isn't that what you mean? Because if I delete then all the data is gone with it. ( I would just delete it and reinstall but I have 25GB downloaded and my Wi-Fi is really slow, this is why I'm troubling with me )
Thank you for taking your time to reply

**Comment by Celarye** [comment]
> Because if I delete then all the data is gone with it.
That's why you save the file on your computer.

> I would just delete it and reinstall but I have 25GB downloaded
You will lose this.

Either way other people had the same issue and this didn't fix it for them so I'm pretty sure it won't for you either. The only other solution is completely starting over.

**Comment by celarye** [comment]
As part of a repository-wide cleanup for the 0.9 release, we are resetting our issue tracker to more efficiently track new feedback and feature requests. If this issue is still relevant or critical to your workflow, please feel free to reopen it with updated context. Thank you for your support!
